### ISSUE TITLE
2.2.0: "This expression is not callable"

### ISSUE THREAD

**Issue by doits** [therealparmesh/object-to-formdata#78]
Since upgrade to 2.2.0 my typescript compiler gives me this error:

```
292:24 This expression is not callable.
  Type 'typeof import(".../node_modules/object-to-formdata/index")' has no call signatures.
    291 |
  > 292 |     const serialized = objectToFormData(params)
```

I imported it like this:

```
import objectToFormData from "object-to-formdata"
```

It still works, but the typescript error is there. Typescript 3.7.3.

**Comment by iangoh94** [comment]
Fixed by version 2.2.1

**Comment by therealparmesh** [comment]
I moved the package away from a default export in version 3.0.0.
